### PR TITLE
feat: combine controllers to one deployment [Release Note]

### PR DESCRIPTION
**Pull request by whynowy**
Closes: #1564

Need to state in release notes:

Before applying new installation yaml, need to run following command to clean up old controller deployments.
```shell
kubectl -n argo-events delete deploy sensor-controller eventsource-controller eventbus-controller --ignore-not-found
```

Signed-off-by: Derek Wang <whynowy@gmail.com>

Checklist:

* [ ] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

### CODE REVIEW

File: controllers/cmd/start.go [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+		logger.Fatalf("required environment variable '%s' not defined", imageEnvVar)
+	}
+	opts := ctrl.Options{
+		MetricsBindAddress:     fmt.Sprintf(":%d", common.ControllerMetricsPort),
+		HealthProbeBindAddress: ":8081",
+	}
+	if namespaced {
+		opts.Namespace = managedNamespace
+	}
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), opts)
```
Comment (sarabala1979) [inline controllers/cmd/start.go:53]: Do we need to merge the config?

**Review by sarabala1979 — APPROVED** [review]
LGTM. Test all edge cases

**Review by daniel-codefresh — APPROVED** [review]
LGTM!!
